docs/gettingstarted/create-parser.md

```diff
@@ -30,69 +27,67 @@ Msg: 2022-04-28T16:16:15.466731-04:00 NTNX-21SM6M510425-B-CVM audispd[32075]: no
 	
 ```
 
-### Wireshark
-Or you can read the logs using Wireshark from the .pcap file.
-From Wireshark go to Statistics->Conversations, then click on 'Follow Stream'.
+### Procure a raw log message using Wireshark
+You can also read the logs using Wireshark from the .pcap file. From Wireshark go to Statistics > Conversations, then click on 'Follow Stream':
 
 ![ws_conversation](../resources/images/ws_conv.png)
-Once you get your stream of messages, copy one of them.
-NOTE: In UDP there usually will not be any message separators.
 
-### Save raw log message in Splunk or archive
+
+### Save your raw log message in Splunk or an archive
+
+Once you get your stream of messages, copy one of them. Note that in UDP there are not usually any message separators.
+
 See [Obtaining "On-the-wire" Raw Events](https://splunk.github.io/splunk-connect-for-syslog/main/troubleshooting/troubleshoot_resources/#obtaining-on-the-wire-raw-events).
 
 ## Create a unit test
-It is recommended to use the existing test case that is the most similar to your use case.
-The naming convention is `test_vendor_product.py`
-Afterwards, you need to make sure that your log is being parsed correctly by creating a test case. 
+To create a unit test, use the existing test case that is most similar to your use case. The naming convention is `test_vendor_product.py`.
+1. Make sure that your log is being parsed correctly by creating a test case. 
```

Comment:
Please see https://splunk.github.io/splunk-connect-for-syslog/2455/gettingstarted/create-parser/ , it needs newlines to render correctly